js/pages/checks/warnings.js

```diff
@@ -1,27 +1,29 @@
-define(['knockout', 'text!./conceptset-warnings.html',
-    'services/CohortDefinition',
+define(['knockout', 'text!./warnings.html',
     './const',
     './utils',
     'atlas-state',
+    './warnings-badge',
     'databindings',
     'faceted-datatable',
     'css!./style.css',
   ],
-  function (ko, view, cohortDefinitionApi, consts, utils, sharedState) {
-
-    function conceptSetWarnings(params){
-      var self = this;
-      this.currentCohortDefinition = sharedState.CohortDefinition.current;
-      self.cohortDefinitionId = this.currentCohortDefinition().id || ko.observable(-1);
-      self.count = params.count || ko.observable();
+  function (ko, view, consts, utils, sharedState) {
```

Comment:
Why not to replace component function with class during such changes

Reply:
fixed